codegen/testserver/generated.go

```diff
@@ -3365,6 +3672,7 @@ func (ec *executionContext) ___Type_fields(ctx context.Context, field graphql.Co
 		Field:  field,
 	}
 	ctx = graphql.WithResolverContext(ctx, rctx)
+	ctx = ec.Tracer.StartFieldExecution(ctx)
```

Comment:
This still wont capture the time spent unpacking args in `field.ArgumentMap`

Reply:
I want to access ResolverContext in StartFieldExecution method.
e.g. https://github.com/mercari/mtc2018-web/blob/feat-refactor-tracer/server/gqlapi/gqlopencensus/tracer.go#L46